numba/core/config.py

```diff
@@ -337,6 +337,16 @@ def avx_default():
         CUDA_ARRAY_INTERFACE_SYNC = _readenv("NUMBA_CUDA_ARRAY_INTERFACE_SYNC",
                                              int, 1)
 
+        # Path of the directory that the CUDA driver libraries are located
+        CUDA_DRIVER = _readenv("NUMBA_CUDA_DRIVER", str, '')
```

Comment:
Does this have the effect, when combined with https://github.com/numba/numba/pull/6844/files#diff-7685977fe7c53dfd319942fe1b5405bcf1d7e3488e00568b02597efcfc65e92cR115-R124 of making pwd the `envpath`, e.g. `envpath = os.path.abspath(envpath)` == `envpath = os.path.abspath('')` == `$pwd`.

Reply:
I don't think so, I think the change you linked to should prevent that, because `if envpath` is not true when `config.CUDA_DRIVER` is `''`. If someone sets it to `'.'` then they could intentionally make the current working directory the env path. (If this doesn't answer your question, could you re-phrase it? I found it a bit hard to be sure what you were asking about)

Reply:
I just interpreted the code entirely incorrectly, ignore this :)